gatsby-node.js

```diff
@@ -1,13 +1,9 @@
-// Controlling your site’s data in the GraphQL data layer.
```

Comment:
You can bring back theses lines because it's helps to quickly jump into the documentation